source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -132,6 +142,78 @@ json TransformerInstance::env(const inja::Arguments& args) const {
   return "";
 }
 
+json TransformerInstance::cluster_metadata_callback(const inja::Arguments& args) const {
+  const std::string& key = args.at(0)->get_ref<const std::string&>();
+  
+  if (!cluster_metadata_) {
+    return "";
+  }
+
+  const ProtobufWkt::Value& value = Envoy::Config::Metadata::metadataValue(cluster_metadata_, SoloHttpFilterNames::get().Transformation, key);
```

Comment:
i think this line wont compile. if that's the case accept the suggestion
```suggestion
  const ProtobufWkt::Value& value = Envoy::Config::Metadata::metadataValue(*cluster_metadata_, SoloHttpFilterNames::get().Transformation, key);
```

Reply:
They very recently changed it:
- the old version took a ref: https://github.com/envoyproxy/envoy/blob/v1.13.1/source/common/config/metadata.h#L39
- the current master (which we are now tracking) takes a pointer: https://github.com/envoyproxy/envoy/blob/master/source/common/config/metadata.h#L45